che-core-vfs-impl/src/main/java/org/eclipse/che/vfs/impl/fs/FSMountPoint.java

```diff
@@ -297,6 +249,11 @@ protected AccessControlList loadValue(Path key) {
     private final VirtualFileSystemUserContext userContext;
 
     /**
+     * ACL that returned for any fs entry.
+     */
+    private final AccessControlList defaultAcl = new AccessControlList();
```

Comment:
I would say that it needs to be static as well as it's immutable and not defined per instance